book/4 - MultiWorlds/4 - Linking Worlds Without Plugins.html

```diff
@@ -0,0 +1,12 @@
+<p>
+	You can link worlds without resorting to plugins by modifying <code>world.ini</code> files.
+	However, this method is limited: Each world can only be linked to 2 different worlds.
+</p>
+
+<p>
+	By default, stepping through any Nether Portal leads to the Nether, and stepping through any End Portal leads to the end.
```

Comment:
I thought that Cuberite had no nether or end by default, can you elaborate here for my sake?